WhichSpace/Classes/AppDelegate.swift

```diff
@@ -47,6 +47,13 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate, SUUpdaterDel
             name: NSNotification.Name("AppleInterfaceThemeChangedNotification"),
             object: nil
         )
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(AppDelegate.updateActiveSpaceNumber),
+            // name: NSWindow.didUpdateNotification,
+            name: NSApplication.didUpdateNotification,
```

Comment:
@tidues Have you had a chance to look at this yet?

Reply:
Sorry. I didn't see this. I'll have a lot and let you know.

Reply:
I remember that before using this notification, when jumping between multiple monitors, sometimes the workspace number indicator did not update (I don't remember the exact conditions for this to happen since it has been a while). So, what I did was printing out all the notifications, and this notification could resolve the issue.